plutus-tx-plugin/src/PlutusTx/Options.hs

```diff
@@ -171,10 +171,12 @@ pluginOptions =
               , Implication (== True) posPreserveLogging True
               , Implication (== True) posCaseOfCaseConservative True
               , Implication (== True) posInlineConstants False
+              , Implication (== True) posDoSimplifierEvaluateBuiltins False
```

Comment:
It gets turned off here because it may increase program size?

Reply:
Because it may change semantics: https://github.com/IntersectMBO/plutus/issues/6167